src/GafferML/Inference.cpp

```diff
@@ -94,7 +119,18 @@ Ort::Session &acquireSession( const std::string &fileName )
 		throw Exception( fmt::format( "Could not find file \"{}\" on GAFFERML_MODEL_PATHS", fileName ) );
 	}
 
-	it = g_map.try_emplace( fileName, acquireEnv(), path.c_str(), Ort::SessionOptions() ).first;
+	auto sessionOpt = Ort::SessionOptions();
+	auto customLibraryPaths = customOpLibraryPaths();
+	for ( const auto& customLibraryPath : customLibraryPaths )
+	{
+		Ort::ThrowOnError(
+			Ort::GetApi().RegisterCustomOpsLibrary_V2(
+				static_cast<OrtSessionOptions*>( sessionOpt ),
+				customLibraryPath.c_str()
+			)
+		);
+	}
+	it = g_map.try_emplace( fileName, acquireEnv(), path.generic_string().c_str(), sessionOpt ).first;
```

Comment:
Was the change from `path.c_str()` to `path.generic_string().c_str()` necessary here, and if so, why? It seems unrelated to the rest of this commit.

Reply:
addressed here 6f5be1b85acd1b28627a9ebdd37ee840820071c9